### PR TITLE
added vpc data

### PR DESCRIPTION
**Pull request by yariv-doit**
removed comments from VPC info

why was this comment out? 
I find these fields useful  
can it be merged to main repository?

### CODE REVIEW

File: examples/instances/instances.go [hunk 1]
```diff
@@ -4,7 +4,7 @@ import (
 	"fmt"
 	"os"
 
-	ec2instancesinfo "github.com/cristim/ec2-instances-info"
+	ec2instancesinfo "github.com/yariv-doit/ec2-instances-info"
```
Comment (cristim) [inline examples/instances/instances.go]: Please keep this out of the commit.

File: go.mod [hunk 2]
```diff
@@ -1,8 +1,5 @@
-module github.com/cristim/ec2-instances-info
+module github.com/yariv-doit/ec2-instances-info
 
 go 1.16
 
-require (
-	github.com/go-bindata/go-bindata v3.1.2+incompatible // indirect
-	github.com/pkg/errors v0.9.1
```
Comment (cristim) [inline go.mod]: And this

File: go.sum [hunk 3]
```diff
@@ -1,4 +1,2 @@
-github.com/go-bindata/go-bindata v3.1.2+incompatible h1:5vjJMVhowQdPzjE1LdxyFF7YFTXg5IgGVW4gBr5IbvE=
-github.com/go-bindata/go-bindata v3.1.2+incompatible/go.mod h1:xK8Dsgwmeed+BBsSy2XTopBn/8uK2HWuGSnA11C3Joo=
```
Comment (cristim) [inline go.sum]: Not sure why this is still shown, we no longer use go-bindata.

**Review by cristim — CHANGES_REQUESTED** [review]
Awesome, I honestly never needed those fields before. There are probably dozens of other fields available in the JSON that we don't expose yet.

There's also a concern on possible memory consumption at runtime if we expose more data, we may need to make it configurable.

File: go.mod (hunk 2 above)
Comment (yariv-doit) [reply go.mod]: not sure I understand, this was removed by `go mod tidy`
you state below "we're not using go-bindata"

**Comment by yariv-doit** [comment]
looks like the test are failing due to go version 
https://github.com/terraform-docs/terraform-docs/issues/474
... [Embedded Files](https://tip.golang.org/doc/go1.16#library-embed) was added in go1.16.

File: examples/instances/instances.go (hunk 1 above)
Comment (yariv-doit) [reply examples/instances/instances.go]: sorry, published locally, didn't notice it came into the PR, fixed

**Comment by yariv-doit** [comment]
Hi @cristim
first of all, I did not have a chance to thank you for your package.
The reason for the change is that we use this package to compare instances and suggest instances from the same family, where CPU and memory need to be the same or above
recently we had issues selecting instances meeting CPU and Mem requirements but lacking network capacity
this affect pod capacity when using K8S (EKS).

Please let me know if you're going to merge or require more changes or info

**Comment by cristim** [comment]
Thanks for the contribution @yariv-doit, glad you find this useful.

Unfortunately I'm really busy with other higher priority work so don't have time to look into this over the next few days but I'll check it out next week.

**Comment by cristim** [comment]
@yariv-doit I updated the Github Action to use newer versions of Go, please rebase this PR to make use of that.

File: go.sum (hunk 3 above)
Comment (cristim) [reply go.sum]: I did a commit that just did `go mod tidy`so this should no longer be needed.

**Comment by cristim** [comment]
oh, i messed up, didn't intend to close this, sorry @yariv-doit.

I was just trying to use this https://docs.github.com/en/repositories/working-with-files/managing-large-files/configuring-git-large-file-storage in order to decrease the size of the repo but the Go tooling breaks with it.

Please rebase your changes against the current HEAD and make another PR.

Sorry for the inconvenience.

**Pull request closed without merging**